public/css/style.css

```diff
@@ -7,17 +7,21 @@ html {
     height: 100%;
 }
 
+
 body {
   padding: 0;
   margin: 0;
   height: 100%;
   background-color: #205f8c;
-}
+  font: 100% Lato, sans-serif;
```

Comment:
I think if you put this font in the body, you don't have to put it in the #footer etc as well